pkg/resources/prometheusRules.go

```diff
@@ -50,7 +50,13 @@ type AlertConfiguration struct {
 func (r *AlertReconcilerImpl) ReconcileAlerts(ctx context.Context, client k8sclient.Client) (integreatlyv1alpha1.StatusPhase, error) {
 	// If the installation was marked for deletion, delete the alerts
 	if r.Installation.DeletionTimestamp != nil {
-		allAlerts := append(r.Alerts, r.RemovedAlerts...)
+		//removing RHOAMUpgradeExpectedDuration30minExceeded alert as it was renamed (30min -> 60min)
+		alertToDelete := AlertConfiguration{
```

Comment:
@valerymo this will only work with when the deletionTimestamp is present. Therefore the alert to be deleted will only be added to the r.RemovedAlerts will only be updated after the deletion of RHOAM is triggered.

Reply:
@MStokluska Fixed, hope better now. Thank you for comment and review